### DISCUSSION TITLE
v1.8.6 - Bug Fixes

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#908] [General]
* Fixed audio break up with some apps on lower sample rates
* Fixed app asking to install driver after every restart

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.8.6'>v1.8.6 - Bug Fixes</a>.</em>

**Comment by lisa-lthorrold** [comment]
Unfortunately, I think this app broke completely on my Mac Mini M2 (Sonoma) - driver install failure loop on startup

Driver install Failed: The operation couldn’t be completed. (Shared.XPCError error 5.)

**Reply by nodeful** [reply]
Have you tried restarting your Mac?
